apps/testingaccessibility/src/pages/learn/index.tsx

```diff
@@ -63,27 +59,16 @@ const productQuery = groq`*[_type == "product" && productId == $productId][0]{
   }`
 
 export const getServerSideProps: GetServerSideProps = async ({req}) => {
-  const sessionToken = await getDecodedToken(req)
+  const {product, purchases} = await getPurchasedProduct(req, productQuery)
```

Comment:
the `productQuery` here was a lot bigger and I didn't know if it should just be the default or if we wanted smaller queries for specific reasons so I made it an optional arg